geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/SimpleFeatureConverterFactory.scala

```diff
@@ -93,6 +104,10 @@ trait SimpleFeatureConverter[I] extends Closeable {
    */
   def processInput(is: Iterator[I], ec: EvaluationContext = createEvaluationContext()): Iterator[SimpleFeature]
 
+  def processSingleInput(i: I, ec: EvaluationContext = createEvaluationContext()): Seq[SimpleFeature]
+
+  def process(is: InputStream, ec: EvaluationContext = createEvaluationContext()): Iterator[SimpleFeature] = ???
```

Comment:
should we make it a required implementation? I would do that, or extend the trait with another trait that has this method.